VAMobile/src/store/slices/authSlice.ts

```diff
@@ -807,6 +808,9 @@ const authSlice = createSlice({
     dispatchFinishAuthLogin: (state, action: PayloadAction<AuthFinishLoginPayload>) => {
       const successfulLogin = !action.payload.error
 
+      if (successfulLogin) {
+        postLoggedIn()
+      }
```

Comment:
Since this is a reducer, we shouldn't be performing any async actions here because this should be a pure function. I'd suggest calling this in the thunks, right before the `dispatchFinishAuthLogin` calls are made

Reply:
updated also added logging errors to the api